samples/99_zil_interpreter/tests/builtins_tests.rb

```diff
@@ -28,8 +28,138 @@ def test_builtin_lval_raises_error_when_not_existing(args, assert)
 def test_builtin_plus(args, assert)
   zil_context = build_zil_context(args)
 
-  # <+ 1 2 <+ 9 10>>
-  result = zil_context.globals[:+].call [1, 2, Syntax::Form.new(:+, 3, 4)], zil_context
+  # <+>
+  result = zil_context.globals[:+].call [], zil_context
+  assert.equal! result, 0
+
+  # <+ 5>
+  result = zil_context.globals[:+].call [5], zil_context
+  assert.equal! result, 5
 
+  # <+ 1 2 <+ 3 4>>
+  result = zil_context.globals[:+].call [1, 2, Syntax::Form.new(:+, 3, 4)], zil_context
   assert.equal! result, 10
 end
+
+def test_builtin_minus(args, assert)
+  zil_context = build_zil_context(args)
+
+  # <- 10 1 <- 5 1>>
+  result = zil_context.globals[:-].call [10, 1, Syntax::Form.new(:-, 5, 1)], zil_context
+  assert.equal! result, 5
+
+  # <- 5>
+  result = zil_context.globals[:-].call [5], zil_context
+  assert.equal! result, -5
+
+  # <- >
+  result = zil_context.globals[:-].call [], zil_context
+  assert.equal! result, 0
+end
+
+def test_builtin_multiply(args, assert)
+  zil_context = build_zil_context(args)
+
+  # <* 1 5>
+  result = zil_context.globals[:*].call [1, 5], zil_context
+  assert.equal! result, 5
+
+  # <* 1 5 <* 5 2>>
+  result = zil_context.globals[:*].call [1, 5, Syntax::Form.new(:*, 5, 2)], zil_context
+  assert.equal! result, 50
+
+  # <* 1>
+  result = zil_context.globals[:*].call [], zil_context
+  assert.equal! result, 1
+end
+
+def test_builtin_divide(args, assert)
+  zil_context = build_zil_context(args)
+
+  # </ 10 </ 50 10>>
+  result = zil_context.globals[:/].call [10, Syntax::Form.new(:/, 50, 10)], zil_context
+  assert.equal! result, 2
+
+  # </ 10 2>
+  result = zil_context.globals[:/].call [10, 2], zil_context
+  assert.equal! result, 5
+
+  # </ 5>
+  result = zil_context.globals[:/].call [5], zil_context
+  assert.equal! result, 0
```

Comment:
I added this.  I'm not 100% sure if I should have bothered with the float logic.  I don't see any floats being used in the zork1/2/3 code, so it might just be adding complexity we don't need. 🤔

Reply:
I *believe* that ZIL itself does not have floating point numbers